exporters/prometheus/exporter.go

```diff
@@ -0,0 +1,185 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build go1.18
+// +build go1.18
```

Comment:
You'll need to add a `doc.go` file to include package documentation. It should not include this build guard, that way Go versions < 1.18 will not fail to build (but silently do nothing).

Reply:
e.g. https://github.com/open-telemetry/opentelemetry-go/blob/new_sdk/main/exporters/otlp/otlpmetric/doc.go